demisto_sdk/commands/find_dependencies/find_dependencies.py

```diff
@@ -35,14 +35,17 @@ def __exit__(self, type_, value, traceback):
         self.fd = None
 
 
-def parse_for_pack_metadata(dependency_graph: nx.DiGraph, graph_root: str) -> tuple:
+def parse_for_pack_metadata(dependency_graph: nx.DiGraph, graph_root: str, complete_data: bool = False,
+                            id_set_data: dict = {}) -> tuple:
```

Comment:
don't use dict as a default value in functions.
```suggestion
                            id_set_data: dict = None) -> tuple:
```

Reply:
This is the way to do that for the `get` later.